app/uk/gov/hmrc/cgtpropertydisposals/models/SubscriptionDisplayResponse.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cgtpropertydisposals.models.address.Address
+import uk.gov.hmrc.cgtpropertydisposals.models.ids.CgtReference
+import uk.gov.hmrc.cgtpropertydisposals.models.name.ContactName
+
+final case class SubscriptionDisplayResponse(
+  emailAddress: Option[Email],
+  address: Address,
+  contactName: ContactName,
+  cgtReference: CgtReference,
+  telephoneNumber: Option[TelephoneNumber],
+  registrationWithId: Boolean
```

Comment:
could we call this `registeredWithId`? the past tense conveys the fact that the registration happened in the past

Reply:
Yes I also noticed that it broke the end to end manual test

Reply:
changed